tests/MongoDB.Driver.Tests/Specifications/unified-test-format/UnifiedTestOperationFactory.cs

```diff
@@ -0,0 +1,210 @@
+                            throw new NotImplementedException();
+                        case "assertCollectionNotExists":
+                            throw new NotImplementedException();
+                        case "assertDifferentLsidOnLastTwoCommands":
+                            throw new NotImplementedException();
+                        case "assertIndexExists":
+                            throw new NotImplementedException();
+                        case "assertIndexNotExists":
+                            throw new NotImplementedException();
+                        case "assertSessionDirty":
+                            throw new NotImplementedException();
+                        case "assertSessionNotDirty":
+                            throw new NotImplementedException();
+                        case "assertSessionPinned":
+                            throw new NotImplementedException();
+                        case "assertSessionUnpinned":
+                            throw new NotImplementedException();
+                        case "assertSameLsidOnLastTwoCommands":
+                            throw new NotImplementedException();
+                        case "assertSessionTransactionState":
+                            throw new NotImplementedException();
+                        case "assertEventCount":
+                            throw new NotImplementedException();
+                        case "configureFailPoint":
+                            throw new NotImplementedException();
+                        case "failPoint":
+                            return new UnifiedFailPointOperationBuilder(_entityMap).Build(operationArguments);
+                        case "recordPrimary":
+                            throw new NotImplementedException();
+                        case "runAdminCommand":
+                            throw new NotImplementedException();
+                        case "runOnThread":
+                            throw new NotImplementedException();
+                        case "startThread":
+                            throw new NotImplementedException();
+                        case "targetedFailPoint":
+                            throw new NotImplementedException();
+                        case "wait":
+                            throw new NotImplementedException();
+                        case "waitForEvent":
+                            throw new NotImplementedException();
+                        case "waitForPrimaryChange":
+                            throw new NotImplementedException();
+                        case "waitForThread":
+                            throw new NotImplementedException();
+                        default:
+                            throw new FormatException($"Invalid method name: \"{operationName}\".");
+                    }
+
+                case var _ when targetEntityId.StartsWith("client"):
+                    switch (operationName)
```

Comment:
is it guaranteed in the spec that all client entities are prefixed with "client"?